i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_nvidia.md

```diff
@@ -0,0 +1,347 @@
+---
```

Comment:
A branch should only resolve a single Issue to prevent the content of different issues from being included in the same PR

Reply:
OKay, thanks for your advice. I changed it.